test/build/Jamfile.v2

```diff
@@ -51,13 +70,18 @@ project wave/test
         cxx11_hdr_mutex
         cxx11_hdr_regex
       ]
+      <include>$(SAMPLES_DIR)
```

Comment:
I'm surprised to see that the tests might depend on the samples (and that this was detected in the process of this work). Can you recall why this is required?

Reply:
I see from later changes that it's `lexertl` :)